### PR TITLE
New relations dialog improvements

### PR DESCRIPTION
**Pull request by Marginy605**

### CODE REVIEW

File: src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts [hunk 1]
```diff
@@ -55,6 +57,8 @@ export type ConnectionsData = Array<{
 export type AliasClickHandlerData = {
     showDebugInfo: boolean;
     currentRow: DashkitMetaDataItem;
+    relationText: React.ReactNode | string;
```
Comment (mournfulCoroner) [inline src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts:60]: Doesn't ReactNode include string?

File: src/ui/units/dash/containers/Dialogs/DialogRelations/components/Content/Row.tsx [hunk 2]
```diff
@@ -158,34 +160,62 @@ export const getTooltipInfo = ({
...
             : undefined;
 
-    return {tooltipContent, tooltipTitle};
+    const aliasDetailTitle = (
+        <React.Fragment>
+            {getRelationDetailsText({
+                text: i18n(getRelationDetailsKey(relationType)),
+                linkType: relationType as RelationType,
+                widget: `${widgetLabel}${widgetSeparator}${widgetTitle}`,
+                row: `${rowLabel}${rowSeparator}${rowTitle}`,
```
Comment (mournfulCoroner) [inline src/ui/units/dash/containers/Dialogs/DialogRelations/components/Content/Row.tsx:207]: Row and widget are identical here and in `tooltipTitle`. Perhaps we should put this in a variable?

File: src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts (hunk 1 above)
Comment (Marginy605) [reply src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts:60]: According to @types react it doesn't.
image

File: src/ui/units/dash/containers/Dialogs/DialogRelations/components/Content/Row.tsx (hunk 2 above)
Comment (Marginy605) [reply src/ui/units/dash/containers/Dialogs/DialogRelations/components/Content/Row.tsx:207]: It is not final variant of texts, I'm sure after design review we will change it not once

File: src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts (hunk 1 above)
Comment (mournfulCoroner) [reply src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts:60]: Well hidden

Comment (Marginy605) [reply src/ui/units/dash/containers/Dialogs/DialogRelations/types.ts]: fixed

**Review by mournfulCoroner — APPROVED** [review]

**Pull request merged**